README.md

```diff
@@ -170,6 +170,31 @@ You can nest properties using `.` as a separator, for example, if you want gener
 npx i18n-populator --text "Hello beautiful world" --from "en" --name "landing_page.greetings"
 ```
 
+## Alphabetical JSON Sorting
+
+This update permanently enables alphabetical sorting of JSON keys during the translation process. By doing so, the translation files will have their keys organized automatically, reducing merge conflicts and improving overall readability.
+
+### What It Does
+
+- **Automatic Sorting:**  
+  When translations are generated, keys in the JSON files will be sorted alphabetically.
+
+- **Reduced Merge Conflicts:**  
+  New translation entries are inserted into the appropriate order, minimizing the chance of conflicts when multiple changes are merged.
+
+### How to Enable
+
+To enable this feature permanently, add `"sort": true` in your `i18n-populator.config.json` file:
+
+```json
+{
+  "basePath": "example",
+  "sort": true,
+  "translationEngines": [ ... ],
+  "languages": [ ... ]
+}
+```
+
```

Comment:
Please create a new markdown file in a `/docs` folder in the root of the project, maybe  `/docs/sort.md`